addons/account/wizard/account_resequence.py

```diff
@@ -133,6 +135,7 @@ def resequence(self):
         if self.move_ids.journal_id and self.move_ids.journal_id.restrict_mode_hash_table:
             if self.ordering == 'date':
                 raise UserError(_('You can not reorder sequence by date when the journal is locked with a hash.'))
+        self.env['account.move'].browse(int(k) for k in new_values.keys()).name = False
```

Comment:
Need to cast as a list to avoid expiring the generator in extensions

Reply:
I don't know what I was on when I wrote the comment about the generator being expired. I cannot since the `browse()` is converting it as a list internally.